permissions/plugin-notify-events.yml

```
@@ -0,0 +1,7 @@
+---
+name: "notify-events"
+github: "jenkinsci/notify-events-plugin"
+paths:
+- "org/jenkins-ci/plugins/notify-events"
```

```suggestion
- "events/notify/notify-events"
```

Needs to be the `groupId` from your pom.xml

---

See https://github.com/jenkinsci/notify-events-plugin/blob/master/pom.xml#L10